nginx-agent-docker/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:7
```

Comment:
Does it need to be CentOS 7? Can't it be a newer release, since CentOS 7 is EOL?